starters/basic-starter/package.json

```diff
@@ -25,12 +25,14 @@
   "devDependencies": {
     "@babel/core": "^7.18.9",
     "@types/node": "^18.0.6",
-    "@types/react": "^17.0.4",
     "autoprefixer": "^10.4.2",
     "eslint": "^8.20.0",
     "eslint-config-next": "^12.2.3",
     "postcss": "^8.4.5",
     "tailwindcss": "^3.0.15",
     "typescript": "^4.6.0"
+  },
+  "resolutions": {
+    "@types/react": "^17.0.4"
```

Comment:
I moved @types/react to here for now because I was getting a compile error in _app.tsx and this fixed it. 

`Component' cannot be used as a JSX component. Its element type 'ReactElement<any, any> | Component<{}, any, any>' is not a valid JSX element.`